package.json

```diff
@@ -71,12 +71,11 @@
   "dependencies": {
     "@babel/core": "^7.18.10",
     "@babel/preset-env": "^7.18.10",
+    "@parse/node-apn": "^5.1.3",
     "acorn": "^8.0.5",
     "acorn-jsx": "^5.3.1",
-    "apn": "^2.2.0",
     "async": "^0.9.2",
     "babel-loader": "^8.2.5",
```

Comment:
Looking at the original PR when the prior APN was added, I also saw the "pem" and "base64url" libraries were added. Those don't seem needed from what I can tell since upgrading the library. 

https://github.com/nightscout/cgm-remote-monitor/pull/5043/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519